### PR TITLE
Prose lint

### PR DESCRIPTION
**Pull request by volodymyrss**

### CODE REVIEW

**Comment by volodymyrss** [comment]
purely technical thing, for typo resistance!

**Review by burnout87 — APPROVED** [review]

File: interfaces.md [hunk 1]
```diff
@@ -57,6 +57,57 @@ In general, backends should be able to control access to themselves by instructi
...
+
+**Roles** will be used by **dispatcher** when filtering queries. **Dispatcher** may associate each query with required **Roles**, or establish filters for more complex request matching (e.g. restricting parameter ranges).
+
+Some **Backends** may declare which roles they require, and **dispatcher** will respect these requests. 
+
+Some **backends** (e.g. integral) will declare that they can operate in role-restricted mode, and will perform additional filtering based on the **Role** information provided.
+
+In this case, a more restricted **Token** can be provided to the backend, to prevent backend gaining entire rights of the user. **Restricted Token** can be obtained (on a special service, or another authority source) in exchange for the User **Token**, specifying the narrowing down of the scope.
+
+
```
[the remark this answers is not in the document]
Comment (volodymyrss) [reply interfaces.md]: ok! but since you approve it, I suppose you mean next time?

[the remark this answers is not in the document]
Comment (volodymyrss) [reply interfaces.md]: this is not the addition of this MR, so I merge.

[the remark this answers is not in the document]
Comment (burnout87) [reply interfaces.md]: I must have made some confusion with the other PR (#47 ) and probably this comment is unnecessary, sorry

**Pull request merged**